### ISSUE TITLE
Camera Orientation issue.

### ISSUE THREAD

**Issue by codal-mpawar** [apivideo/api.video-reactnative-live-stream#57]
**Describe the bug**
When we do live stream in landscape mode always displays in portrait mode. even my device/UI orientation is in the landscape.

**To Reproduce**
1. Open the app.
2. Rotate your device in landscape mode.
3. Start streaming.

**Expected behavior**
If the device orientation is in landscape always shows stream in landscape mode.

**Smartphone (please complete the following information):**
 - Device: iPhone 11 pro max
 - iPhone OS Version: 16.2
 - Xcode version 14.2
 - Mac OS: Monterey 12.6

**Additional context**
"@api.video/react-native-livestream": "v1.2.2"
"react": "16.13.1",
"react-native": "0.64.2",
"react-native-orientation-locker": "^1.4.0",

**Comment by ThibaultBee** [comment]
Yes, you are right!
I am going to fix this quickly.

**Comment by ThibaultBee** [comment]
Could you check on the `main` branch that the issue is fixed?

**Comment by codal-mpawar** [comment]
Thanks will check and update you on the same.

**Comment by codal-mpawar** [comment]
@ThibaultBee 
I am not getting updated code. Have you deployed on npm?

**Comment by ThibaultBee** [comment]
No, it is just on `main` branch for now.

**Comment by codal-mpawar** [comment]
Okay, will check and update you on the same.

**Comment by ThibaultBee** [comment]
Ok, the release has been deployed to npm.
Please, reopen this issue if you still face it.

**Comment by codal-mpawar** [comment]
@ThibaultBee Yesterday we check with your main branch and it's working fine but today it's not working. Let me know still if you need more info

"@api.video/react-native-livestream": "^1.2.3"

**Comment by ThibaultBee** [comment]
Hi,
In the code, there are no differences between the main branch and npm release.
Are you sure your Podfile.lock are up to date?

**Comment by codal-mpawar** [comment]
@ThibaultBee Yes it's up to date and also we will delete the pod folder and Podfile.lock.

**Comment by codal-mpawar** [comment]
@ThibaultBee Please check the recording once.
Uploading RPReplay_Final1674552426 (2).mp4…

**Comment by ThibaultBee** [comment]
I don't understand what the video is supposed to show.
I have the feeling that this is another issue.
Could you try to describe the steps so I can reproduce?

**Comment by codal-mpawar** [comment]
Please look at the screen UI that is landscape but the camera is still showing in portrait. that only we need to show in the video.

**Comment by codal-mpawar** [comment]
@ThibaultBee 

1. Open the app
2. before starting the live stream rotate your device in landscape.
3. start a live stream.

**Comment by codal-mpawar** [comment]
@ThibaultBee any update? We are block for release.

**Comment by codal-mpawar** [comment]
@ThibaultBee Any way to resolve the camera mirroring issue?

**Comment by ThibaultBee** [comment]
I don't reproduce the issue on my side.
Could you test with the example in this project?
Could you sure the content of the `Podfile.lock`?

**Comment by codal-mpawar** [comment]
@ThibaultBee Camera orientation works fine.
I have another query
How to resolve mirror mode on iOS.
